webpack.config.js

```diff
@@ -1,18 +1,21 @@
 var path = require('path');
 var webpack = require('webpack');
 
-module.exports = {
-    mode: "development",
+module.exports = (env, argv) => ({
+    //mode: "development",
     plugins: [
-      // http://stackoverflow.com/questions/29080148/expose-jquery-to-real-window-object-with-webpack
-      new webpack.ProvidePlugin({
-        // Automtically detect jQuery and $ as free var in modules
-        // and inject the jquery library
-        // This is required by many jquery plugins
-        jquery: "jquery",
-        jQuery: "jquery",
-        $: "jquery"
-      }),
+        new webpack.DefinePlugin({
+            'process.env.NODE_ENV': argv.mode === 'development' ? JSON.stringify('development') : JSON.stringify('production')
```

Comment:
You don't need the extra JSON.stringify() calls. All it does is make sure it's double-quoted.